docs/configuration/sinks/YandexMessenger.rst

```diff
@@ -0,0 +1,50 @@
+Yandex Messenger
+#################
+
+Robusta can report issues and events in your Kubernetes cluster to Yandex Messenger chats, channels or private conversations.
+
+.. note::
+
+    Tables (alert labels, events etc.) are sent as file attachments because they are not rendered properly in Yandex Messenger clients.
+
+    2-way interactivity (``CallbackBlock``) is not implemented in this sink.
+
+Obtaining your Bot token
+------------------------------------------------
+Follow the quickstart guide in the Yandex Messenger `docs <https://botapi.messenger.yandex.net/docs/#quickstart-guide>`_ to create a new Yandex Messenger bot, and obtain its OAuth token.
+
+Adding your Bot to your chat/channel and obtaining the chat ID
+------------------------------------------------
+Before using the sink, you must add your Bot to the target chat or channel as an administrator.
+
+You also need to get the chat ID. It has the following format ``<integer>/<integer>/<uuid>`` and can be obtained by opening the chat or channel in the web version of Yandex Messenger and url-decoding the last fragment of the page URL path.
+
```

Comment:
Could you share screenshots of the Text, file and image shares so that we can display in the docs?

Reply:
cc @NeonSludge

Reply:
I've added some examples of how the attachments might look in a YM chat or a channel.